sites/partners/src/lib/hooks.ts

```diff
@@ -523,20 +534,31 @@ export const useApplicationsExport = (listingId: string, includeDemographics: bo
       }),
     `applications-${listingId}-${createDateStringFromNow()}.csv`
   )
+
+  return { onExport: res.onExport, exportLoading: res.csvExportLoading }
 }
 
-export const useLotteryExport = (listingId: string, includeDemographics: boolean) => {
-  const { lotteryService } = useContext(AuthContext)
+export const useSpreadsheetExport = (
+  listingId: string,
+  includeDemographics: boolean,
+  forLottery: boolean
+) => {
+  const { applicationsService, lotteryService } = useContext(AuthContext)
   const [exportLoading, setExportLoading] = useState(false)
   const { addToast } = useContext(MessageContext)
 
   const onExport = useCallback(async () => {
     setExportLoading(true)
     try {
-      const content = await lotteryService.lotteryResults(
-        { id: listingId, includeDemographics },
-        { responseType: "arraybuffer" }
-      )
+      const content = forLottery
```

Comment:
nit: I know this isn't part of this PR, but I just noticed it now. the download file name called `${listingId}-${dateString}-test.zip` probably should be something more specific. Instead of "test" we could have it be `applications` or `lottery` depending on `forLottery` boolean

Reply:
nice catch! updated